src/UserService.ts

```diff
@@ -193,6 +193,11 @@ export type SubscriptionSeat = {
    * Is the seat active
    */
   active: boolean;
+
+  /**
+   * Subscription used offline
+   */
+  offline: boolean | null;
```

Comment:
Is this props possible to be null? Will be great we keep type as only Boolean so avoid typescript nullishl complains

Reply:
Checked with backend. The default value is false, so it will not be null. Fixed.